test/fixtures/minimal/main.tf

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider google {
+  project = var.project_id
+  region  = var.region
+}
+
+provider "random" {
+  version = "~> 2.0"
+}
+
+resource "random_string" "suffix" {
+  length  = 8
+  upper   = false
+  lower   = true
+  number  = true
+  special = false
+}
+
+locals {
+  random_suffix = random_string.suffix.result
+  resource_name = "ilb-minimal-${local.random_suffix}"
+  health_check = {
+    type                = "http"
+    check_interval_sec  = 1
+    healthy_threshold   = 4
+    timeout_sec         = 1
+    unhealthy_threshold = 5
+    response            = "I AM HEALTHY"
+    proxy_header        = "NONE"
+    port                = 8081
+    port_name           = "health-check-port"
+    request_path        = "/mypath"
+    host                = "1.2.3.4"
+  }
+}
+
+resource "google_project_service" "test" {
+  project            = var.project_id
+  service            = "compute.googleapis.com"
+  disable_on_destroy = "false"
+}
+
+resource "google_compute_network" "test" {
```

Comment:
Networks and subnetworks should be created in the example.

Reply:
Should I create a separate `minimal` example folder and call it from the test suite?
Right now the `minimal` test suite calls the root module directly. There's a `simple` example module which is not used in tests.

Reply:
Ok, created the separate `minimal` example module.